### ISSUE TITLE
Upgrade Process for Twig/Twig 

### ISSUE THREAD

**Issue by aweingarten** [acquia/blt#3449]
Yesterday a symfony security vulnerability was reported for twig/twig.  This blocked our builds.  We tried to update twig/twig.  Unfortunately there was an api-breaking change in twig/twig.  Drupal core is rolling a patch to fix.  However it looks like its going to be implemented in a way that is incompatible with BLTs composer workflow.

See here:
https://www.drupal.org/project/drupal/issues/3039408

I have failed to apply the patch via composer, because its looking to patch a non-existent lock file.  I'm wondering if anyone has a good idea how to handle this in BLT?  The best idea I have come up with is to hack the patch and remove the lockfile changes

**Comment by aweingarten** [comment]
I should add that the official recommendation in the core issue of manually patching with with git and `--exclude=composer.lock`

then running `composer update twig/twig`
Is not a sustainable workflow for this population.  It breaks our patching process.  

The only ideas I can come up with are
1: Spin a new patch for us 
2: Update cweagans/composer-patches to allow individual files to be excluded

**Comment by mikemadison13** [comment]
Theoretically this is resolved by https://www.drupal.org/project/drupal/releases/8.6.11 but I haven't had a chance to test it out yet.

**Comment by mikemadison13** [comment]
However, updating to 8.6.11 may be blocked by https://www.drupal.org/project/lightning_core/issues/3040013

**Comment by mikemadison13** [comment]
this seems resolved with the Drupal 8.6.11 update and the new lightning_core release. I'm going to close, happy to re-open if other issues arise. 

after updating to Drupal 8.6.11 i am seeing:

```
composer show twig/twig
name     : twig/twig
descrip. : Twig, the flexible, fast, and secure template language for PHP
keywords : templating
versions : * v1.38.2
```

**Comment by mikemadison13** [comment]
FYI there is still a known fatal with certain twig templates in 8.6.11 -- https://www.drupal.org/project/drupal/issues/3040210
